src/api/libcellml/component.h

```diff
@@ -125,13 +125,20 @@ class LIBCELLML_EXPORT Component: public ComponentEntity, public ImportedEntity
     /**
      * @brief Add a variable by reference as part of this component.
      *
-     * Add a variable by reference as part of the given component.
+     * Add a variable by reference as part of the given component. If the variable
+     * already belongs to another component, it is moved to this one, and the
+     * previous component's variable list is updated.
+     *
+     * The function will return @c false and no action is taken if:
+     *  - The @p variable reference already exists in this component; and
+     *  - The @p variable is the @c nullptr.
```

Comment:
Be consistent in our API documentation (see `addReset()` for instance): different wording.

Reply:
d9b0052d